src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/ResultSubjectMetaServiceTests.cs

```diff
@@ -1678,6 +1681,377 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbC
             }
         }
 
+        [Fact]
+        public async Task GetSubjectMeta_LocationsAreDeduplicated_Flat()
+        {
+            var publication = new Publication();
+
+            var subject = new Subject
+            {
+                Id = Guid.NewGuid()
+            };
+
+            var observations = new List<Observation>();
+
+            var releaseId = Guid.NewGuid();
+
+            var locations = new Dictionary<GeographicLevel, List<LocationAttributeNode>>
+            {
+                {
+                    GeographicLevel.LocalAuthority,
+                    new List<LocationAttributeNode>
+                    {
+                        new(_derby),
+                        new(_derbyDupe),
+                        new(_nottingham)
+                    }
+                }
+            };
+
+            // Hierarchies are enabled, but none have been defined
+            // so locations should still all be flat.
+            var options = Options.Create(new LocationsOptions
```

Comment:
Good point, but yeh thought it'd be a little clearer to be explicit in this case. I've moved the comment directly next to the `TableResultLocationHierarchiesEnabled` so that it's more obvious we need to change it when we remove the flag later on.

Reply:
Great, I see the change now, thanks